.changelogs/issue_190.yml

```diff
@@ -0,0 +1,7 @@
+significance: patch
+type: fixed
+links:
+  - "#190"
+attributions:
+  - "@seothemes"
```

Comment:
```suggestion
```
Attributions is for external contributors only :D

Reply:
:D ohkay